Rubberduck.Parsing/Symbols/ArgumentReference.cs

```diff
@@ -45,5 +49,15 @@ public class ArgumentReference : IdentifierReference
         public int NumberOfArguments { get; }
         public VBAParser.ArgumentListContext ArgumentListContext { get; }
         public Selection ArgumentListSelection { get; }
+
+        public override (string context, Selection highlight) HighligthSelection(ICodeModule module)
+        {
+            var lines = module.GetLines(Selection.StartLine, Selection.LineCount).Split('\n');
+
+            var line = lines[0]; // TODO think of something that makes sense for multiline
```

Comment:
Do all cases end in a ` _`? If there could be a case that _doesn't_ ends with a ` _`,  maybe at least add an `...` to alert that it's truncated? 

A proper solution would be to dynamically resize the height of the row for every newline but that might be too much work and out of hte scope for this PR.

Reply:
I like adding `...` to denote truncation, consider it done! The cells wrap content; I think we could fit multiline no problem (except 20 lines might look weird), but here I was heading towards putting everything on one line (stripping line continuations) and coming up with the correct highlight selection. I'll try truncating after the 2nd line, and handle highlighting across two lines -- and more than that will have an ellipsis.

Reply:
Actually I'll leave multiline highlighting an unhandled edge case here, and add an ellipsis given a multiline selection (*and* on truncation, if it's longer than what I'll make a 255 character cap); in principle it's possible for a search result to not show & highlight the searched thing.

Reply:
Okay so it's a bit of a hack, but it works:

![image](https://user-images.githubusercontent.com/5751684/116012315-d260da80-a5f7-11eb-98f6-6f33a5321098.png)

I'm special-casing the case where a skipped argument is the first character on the line (i.e. the whitespace is being considered part of the indent and trimmed as such) to get a highlight for `[ ], "Title"`. It doesn't get an ellipsis because the context isn't actually on multiple lines.

So when the context *does* span multiple lines, I'm only showing the first part of it with ellipsis, and all of it is highlighted:

![image](https://user-images.githubusercontent.com/5751684/116012596-65e6db00-a5f9-11eb-8369-c4f5f7dfe273.png)